test/sync/src/commonMain/kotlin/io/realm/test/mongodb/TestApp.kt

```diff
@@ -99,6 +99,7 @@ class TestApp(
         RealmInterop.realm_clear_cached_apps()
 
         // Delete metadata Realm files
+        // TODO This actually wipes metadata for all realms and not only the specific instance data
```

Comment:
This is intentional, so not sure there is a TODO here?

Reply:
I removed the TODO as it didn't have anything to do with this PR ... but to me it seems like we could potentially wipe files for another app if a test initialized two. Maybe @edualonso can elaborate on if it does the correct thing.